content/blog/2019-12-03-mark-padgham.md

```diff
@@ -0,0 +1,76 @@
+
+{{< figure src = "/img/team/mark-padgham.jpg" width = "250" alt = "Headshot of Mark Padgham" >}}
+
+> I’ve got a lot of concrete visions for extending testing well beyond anything that’s been done in a computational or quantitative way. Convincing leaders in statistics and statistical software that’s a good idea is certainly going to be challenging and very rewarding.
+
+We talked with Mark to learn more about him, and his work.
+
+
+### What do you want people to know about your background?
+
+I live in Germany, but grew up in Australia. I have an undergraduate degree in physics and maths, a PhD in ecology, qualifications in climatology and meteorology, as well as official teaching qualifications. All of these were obtained from universities in Melbourne, Australia - each one from a different university. Following my PhD, I worked in post-doctoral academic positions at the interface of ecology and climatology, before changing country and continent to Germany, and changing academic direction to the study of urban systems.
+
+I have always said I’m dedicated to changing the world one walking trip or one bike trip at a time to change the way people live life in cities, but now I have a much more “meta” personal motivation. I develop software in the hope that it will make the world a better place. It’s focussed on a particular kind of change. My software is primarily motivated by a desire to make the understanding of cities easier and more accessible - from accessing the raw data of street networks (osmdata), to fine-scaled data on mobility (bikedata), to understanding and mapping how, where, and when people move through cities (through the [dodgr](https://github.com/ATFutures/dodgr) package).
+
+
+### What is your favorite thing about R?
+
+I like to use R (rather than another mainstream language) because R is the best language around to develop master control systems from your raw data input through to documentation outputs, as websites, published reports, dashboards and every step in between. They can all be controlled from the one language. There’s no other language that reaches that level of master control system. It is an accolade of what RStudio has done to transform the utility of the language.
+
+When I sit down with R, the first question I ask myself is, how do I get into C++ from here?
+
+
+### What’s an R thing you’d like to learn?
+
+Yeah…lots of things [_long pause while Mark tries to choose one thing_]. To really understand how the C-level language of R interfaces with objects compile from other languages and environments - that would be really cool. And at the level of C code, how lazy-evaluation works. Everyone’s gotten so used to that but you don’t even have to think about it. That’s a piece of amazing magic under the hood that I don’t understand.
```

Comment:
"R interfaces with objects compile**d** from other languages and environments" (typo)